magprime/__init__.py

```diff
@@ -42,6 +42,11 @@ class Attendee:
     def donation_cost(self):
         return self.extra_donation if self.extra_donation else 0
 
```

Comment:
Sure, I've starred this in my inbox and I'll be sure to do that before I delete it.